Ugly hack to include .pc file for both installed static archives for libraw.
LIBDIR and VERSION can be patched in the ebuild. If libraw is built with lcms
support, lcms should be added to Requires and -llcms should be added to Libs.
If libraw is built with OpenMP support, -fopenmp should be added to Cflags and
 -lgopenmp should be added to Libs.

--- a/Makefile
+++ b/Makefile
@@ -20,6 +20,7 @@ all_samples: bin/raw-identify bin/simple_dcraw  bin/dcraw_emu bin/dcraw_half bin
 install: library
 	@if [ -d /usr/local/include ] ; then cp -R libraw /usr/local/include/ ; else echo 'no /usr/local/include' ; fi
 	@if [ -d /usr/local/lib ] ; then cp lib/libraw.a lib/libraw_r.a /usr/local/lib/ ; else echo 'no /usr/local/lib' ; fi
+	@if [ -d /usr/local/lib/pkgconfig ] ; then cp libraw.pc libraw_r.pc /usr/local/lib/pkgconfig ; else echo 'no /usr/local/lib/pkgconfig' ; fi
 
 install-binaries: all_samples
 	@if [ -d /usr/local/bin ] ; then cp bin/[a-z]* /usr/local/bin/ ; else echo 'no /usr/local/bin' ; fi
--- /dev/null
+++ b/libraw.pc
@@ -0,0 +1,11 @@
+prefix=/usr
+exec_prefix=${prefix}
+libdir=${prefix}/LIBDIR
+includedir=${prefix}/include
+
+Name: libraw
+Description: Raw image decoder library (non-thread-safe)
+Requires: 
+Version: VERSION
+Libs: -L${libdir} -lraw -lstdc++
+Cflags: -I${includedir}/libraw
--- /dev/null
+++ b/libraw_r.pc
@@ -0,0 +1,11 @@
+prefix=/usr
+exec_prefix=${prefix}
+libdir=${prefix}/LIBDIR
+includedir=${prefix}/include
+
+Name: libraw
+Description: Raw image decoder library (thread-safe)
+Requires: 
+Version: VERSION
+Libs: -L${libdir} -lraw_r -lstdc++
+Cflags: -I${includedir}/libraw
